recipes/cyvcf2/meta.yaml

```diff
@@ -25,8 +25,8 @@ requirements:
     - numpy
     - zlib
     - libcurl
-    - htslib >=1.9,<2.0
-    - openssl >=1.1.1b,<1.1.2a
+    - htslib ==1.9
```

Comment:
FYI, we only build against htslib 1.9 at the moment.

Reply:
Should I just keep it as `- htslib`? 

cyvcf2 originally only links to 1.9, so I just wanted to prevent it from accidentally link to 1.10+ if bioconda bumps the version in future.

Reply:
Can you make a `conda_build_config.yaml` in the same directory as the cyvcf2 meta.yaml file in it? That can then have the following as content:

    # Version is only compatible with htslib 1.9
    htslib:
      - 1.9

Then make a note of this where the version is specified in the meta.yaml so we can be sure to check with each new version of cyvcf2.

Reply:
I implemented your suggestions. Thank you!